src/components/ReviewFunctionBar.js

```diff
@@ -1,60 +1,38 @@
 import React from "react";
+import Dropdown from "./Dropdown";
 
 class ReviewFunctionBar extends React.Component {
 
     render() {
-        const sessionListItems = this.props.sessionNames.map((name) =>
-            <button className="dropdown-item" key={name} onClick={() => this.props.selectFilter("session", name)}>{name}</button>
-        );
-        const groupButtons = [...Array(7).keys()].map((number) =>
-            <button className="dropdown-item" key={number} onClick={() => this.props.selectFilter("group", number)}>Group {number}</button>
-        );
+        const filterListItems = [
+            { text: "Used", onClick: () => this.props.selectFilter("Used", "") },
+            { text: "Unused", onClick: () => this.props.selectFilter("Unused", "") },
+            { text: "New", onClick: () => this.props.selectFilter("New", "") },
+            { text: "Basic", onClick: () => this.props.selectFilter("Basic", "") },
+            { text: "Plus", onClick: () => this.props.selectFilter("Plus", "") }
+        ]
+
+        const sessionListItems = this.props.sessionNames.map((name) => ({ text: name, onClick: () => this.props.selectFilter("session", name) }));
+        const groupListItems = [...Array(7).keys()].map((number) => ({ text: `Group ${number}`, onClick: () => this.props.selectFilter("group", number) }));
```

Comment:
What is the `[...Array(7).keys()]` doing?

Reply:
It's generating the numbers 0-6 so the buttons are called "Group 0", "Group 1", etc, as dad didn't give me names for the group, just numbers